src/Spatial/Euclidean/Ray3D.cs

```diff
@@ -98,6 +98,7 @@ public static Ray3D Parse(string point, string direction)
         /// </summary>
         /// <param name="s">a string representing the ray</param>
         /// <returns>a ray</returns>
+        [Obsolete("Should not have been made public. Mode Obsolete 2017-12-09")]
```

Comment:
I'm not sure it must be obsoleted, parsing can be nice in unit tests.

Reply:
it calls an obsolete method - isn't it enough to construct the fundamental types (points, vectors, etc) for the unittests without having another custom non standard string format?

Reply:
ok, merging it